topics/single-cell/tutorials/scrna-case-cell-annotation/slides.html

```diff
@@ -36,11 +36,11 @@
 
 ### What is Cell Annotation?
 
-- Categorising cells into cell type categories based on transcriptomic data
+- Classify cells in your data into different cell types based on gene expression data
 
 - Can be done manually or automatically
 
-- Is possible due to single-cell sequencing technology
+- Single-cell sequencing technology provides higher resolution than bulk RNA-seq
```

Comment:
I feel like this point is missing something to explicitly link it to the cell annotation - maybe combine it with the start of the old version like 'Is possible because single-cell seq tech provides higher res...'?